### PR TITLE
Prepare Barbarian Assault plugin for changes passed in the recent poll

### PR DESCRIPTION
**Pull request by Jacoblairm**
Change the role identifier from cape slot "Icons" to Role Horns in the inventory. In preparation for the recently passed poll question - allowing capes in Barbarian Assault. https://i.imgur.com/45vJHNe.png

Also null the gameTime after a game has complete to fix a minor bug causing the game duration announcement after completing the tutorial.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java [hunk 1]
```diff
@@ -152,20 +154,32 @@ public void onGameTick(GameTick event)
 			return;
 		}
 
-		switch (client.getLocalPlayer().getPlayerComposition().getEquipmentId(KitType.CAPE))
+		for (Item item : client.getItemContainer(InventoryID.INVENTORY).getItems())
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java]: This should probably be put into a new method `private Role getRole(ItemContainer itemContainer)`, and can definitely be removed from `onGameTick` to run instead on item container change.

File: runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java [hunk 2]
```diff
@@ -120,6 +121,7 @@ public void onWidgetLoaded(WidgetLoaded event)
 			if (config.waveTimes() && rewardWidget != null && rewardWidget.getText().contains(ENDGAME_REWARD_NEEDLE_TEXT) && gameTime != null)
 			{
 				announceTime("Game finished, duration: ", gameTime.getTime(false));
+				gameTime = null;
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java]: This is a bug and should be in a separate commit.

**Comment by Adam-** [comment]
I'd rather wait and see how they do the team stuff. I don't think they can assign players to teams without them weilding items associated to those teams?

**Comment by ShaunDreclin** [comment]
iirc ash said they were going to do some magic with beards instead of taking up any of the equipment slots

**Comment by Nightfirecat** [comment]
Is this PR more correct, or is #8955 more correct?

**Comment by Jacoblairm** [comment]
#8955

**Comment by Nightfirecat** [comment]
I'll close this for now then. Re-open if this is still needed in addition to 8955.
